### PR TITLE
Fix bug where waiting for others to join text shows up in black under dark mode, and showing up in screen share

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Fix bug where waiting for others to join text shows up in black under dark mode, and showing up in screen share

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/LocalVideoTile.tsx [hunk 1]
```diff
@@ -249,7 +252,9 @@ export const _LocalVideoTile = React.memo(
           displayName={displayName}
           initialsName={initialsName}
           styles={videoTileStyles}
-          onRenderPlaceholder={props.participantsCount === 1 ? onRenderAvatarOneParticipant : onRenderAvatar}
+          onRenderPlaceholder={
```
Comment (JamesBurnside) [inline packages/react-components/src/components/LocalVideoTile.tsx:255]: Is there a reason this lives in the video tile? For the lobby/connecting overlay messages its just over the gallery, I'd assume there would be logic we would be reusing for both?

Comment (carocao-msft) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: The UI is different when camera in on/ off. When camera is on it's a white text and when camera is off, it's an overlay and need to remove the avatar

Comment (JamesBurnside) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: We need to know about camera status yes, but as a UI component - why do we show a message that's very specific to the composite experience? We shouldn't assume everytime a localvideotile is showing and alone in a call that that translates to "waiting for others to join a call".

Can this be moved to the composite layer? And reuse the lobby logic? This keeps styling etc. consistent. The screens are identical except for the text message:

| | lobby | call (alone) |
| -- | -- | -- |
| no camera | image | image |
| camera on | image | image |

Comment (JamesBurnside) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: Hmm just looking at the code that could be tricky, the lobby isn't using the videogallery but instead just a single tile...

Comment (JamesBurnside) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: Can we still move the logic to the composite? Unless you feel this should be the default experience for everyone using the video gallery?

Comment (carocao-msft) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: This is what Ezra requested so I think component and composite should be consistent? It's overall an UI improvement to show the waiting text when there is only one person in the call

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/components/LocalVideoTile.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-components/src/components/LocalVideoTile.tsx:255]: Chatted offline, this is already in stable so we can't move. If there's a reason someone wishes this to not be shown we can add a new boolean flag to disable

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**